packages/toolkit/src/view/model/ModelsTable.tsx

```diff
@@ -61,7 +61,7 @@ export const ModelsTable = (props: ModelsTableProps) => {
               secondaryLink={null}
               secondaryText={label}
               iconElement={getIcon(
-                `w-4 h-4 ${["TASK_CHAT"].includes(row.original.task || "") ? "stroke-semantic-fg-primary [&>*]:!stroke-semantic-fg-primary" : "[&>*]:!fill-semantic-fg-primary"}`,
+                `w-4 h-4 ${["TASK_CHAT", "TASK_CUSTOM"].includes(row.original.task || "") ? "stroke-semantic-fg-primary [&>*]:!stroke-semantic-fg-primary" : "[&>*]:!fill-semantic-fg-primary"}`,
```

Comment:
@orangecoloured We can use cn here 

cn("w-4 h-4", .......) it will be easier to read compare to template string